nvflare/apis/job_def.py

```diff
@@ -108,14 +111,14 @@ def get_deployment(self) -> Dict[str, List[str]]:
         """
         return self.deploy_map
 
-    def get_application(self, participant, fl_ctx: FLContext) -> bytes:
+    def get_application(self, app_name, fl_ctx: FLContext) -> bytes:
```

Comment:
Since the Job class means only the job config, I think this method should not be in this class.